### PR TITLE
WheelEvent ref - fix up to use specifications table

### PR DESCRIPTION
**Pull request by hamishwillee**
This replaces the manual spec table with one for BCD for WheelEvent. I'm going to be doing work on this page possibly for FF90, so this is cleanup prior to real work :-)

Note, tested - the BCD data works and all of these spec tables render. Rubber stamp review required.

Related to #5385

### CODE REVIEW

File: files/en-us/web/api/wheelevent/deltamode/index.html [hunk 1]
```diff
@@ -53,23 +52,7 @@ <h2 id="Example">Example</h2>
 console.log(syntheticEvent.deltaMode);
 </pre>
 
-<h2 id="Specifications">Specifications</h2>
```
Comment (wbamberg) [inline files/en-us/web/api/wheelevent/deltamode/index.html:56]: The heading needs to be reinstated  here.

**Review by wbamberg — CHANGES_REQUESTED** [review]
One of the deletions was a bit over-zealous.

File: files/en-us/web/api/wheelevent/deltamode/index.html (hunk 1 above)
Comment (hamishwillee) [reply files/en-us/web/api/wheelevent/deltamode/index.html:56]: Thanks. Done.

**Review by wbamberg — APPROVED** [review]
👍 thanks @hamishwillee !

**Pull request merged**